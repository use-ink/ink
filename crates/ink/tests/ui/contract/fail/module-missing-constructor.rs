#![allow(unexpected_cfgs)]

#[ink::contract]
mod contract {
    #[ink(storage)]
    pub struct Contract {}

    impl Contract {
        #[ink(message)]
        pub fn message(&self) {}
    }
}

fn main() {}
