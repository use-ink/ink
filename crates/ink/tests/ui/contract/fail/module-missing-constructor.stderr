error: missing ink! constructor
  --> tests/ui/contract/fail/module-missing-constructor.rs:2:1
   |
2  | / mod contract {
3  | |     #[ink(storage)]
4  | |     pub struct Contract {}
...  |
9  | |     }
10 | | }
   | |_^
